src/workloads/query/CollScanPredicateSelectivityMedium.yml

```diff
@@ -0,0 +1,26 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of conjunctive collection scan queries where the order of
+  predicates matters due to selectivity of the predicates.
+
+LoadConfig:
+  Path: ../../phases/query/CollScanPredicateSelectivity.yml
+  Parameters:
+    Database: CollScanPredicateSelectivityMedium
+    DocumentCount: 1e4
+    Repeat: 100
```

Comment:
Is the repeat significant in terms of a constant _DocumenCount_ * _Repeat_ between the workloads? And if so then should it be 1_000 (workload time permitting).